fe/fe-core/src/main/java/org/apache/doris/catalog/Database.java

```diff
@@ -604,6 +618,10 @@ public void readFields(DataInput in) throws IOException {
         if (Env.getCurrentEnvJournalVersion() >= FeMetaVersion.VERSION_105) {
             dbProperties = DatabaseProperty.read(in);
         }
+
+        if (Env.getCurrentEnvJournalVersion() >= FeMetaVersion.VERSION_116) {
+            transactionQuotaSize = in.readLong();
+        }
```

Comment:
There should be an `else` to set `transactionQuotaSize` to Config.defaut_xxx;

Reply:
Done